Contest_API.md

```diff
@@ -686,7 +686,7 @@ The set of properties listed must always support
 [referential integrity](#referential-integrity), i.e. if a property with a ID 
 value referring to some type of object is present the endpoint representing
 that type of object (and its ID property) must also be present. E.g. if 
-`group_ids` is listed among the properties in the `team` endpoint object, that
+`groups` is listed among the properties in the `team` endpoint object, that
```

Comment:
Sorry to nitpick, but I think this would be better reverted: we actually do have a property `group_ids` in the teams endpoint, but not `groups`. And in general that format `<type>_id(s)` is how we reference other endpoint IDs in the specification.